src/mewbot/tools/lint.py

```diff
@@ -167,7 +167,12 @@ def lint_pylint(self) -> Iterable[Annotation]:
 
         result = self.run_tool("PyLint", "pylint")
 
-        for line in result.stdout.decode("utf-8").split("\n"):
+        try:
+            result_lines = result.stdout.decode("utf-8")
+        except UnicodeError:
+            result_lines = result.stdout.decode("utf-8", errors="replace")
```

Comment:
Is there a semantic difference to having the try catch as opposed to seeing this by default?

Reply:
It _did_, before I spent half an hour trying different error handling methods, and decided replace was best.... corrected.